deploy/cluster_role_createns.yaml

```diff
@@ -0,0 +1,32 @@
+#
+#  Copyright (c) 2012-2019 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: che-create-namespaces
+  labels:
+    app: che
```

Comment:
Is it a role that is to be applied to the operator it self (=> also integrated into the OLM CSV)  or the the Che server when it is created by the Che operator.
To me *only the permissions granted to the operator itself* should be set by such yaml files.
On the contrary the *permissions granted by the operator to the Che server* (or other K8S objects created by the operator from a `CheCluster)` should be created / bound by the operator when installing a Che server.

Reply:
file removed

Reply:
Thanks @AndrienkoAleksandr !